app/controllers/projects_controller.rb

```diff
@@ -67,6 +72,13 @@ def destroy
     end
   end
 
+  def issues_list
```

Comment:
Why did we create a new action for issues_list? What is wrong with having a show.js.erb (Along with html.erb) to handle the js case?